internal/controller/controllers/clusterdeployments_controller.go

```diff
@@ -566,6 +566,13 @@ func isUserManagedNetwork(clusterInstall *hiveext.AgentClusterInstall) bool {
 	return false
 }
 
+func isDiskEncryptionEnabled(clusterInstall *hiveext.AgentClusterInstall) bool {
+	return clusterInstall.Spec.DiskEncryption != nil &&
```

Comment:
Just for readability, maybe we could early return if nil, and use switch on DiskEncryption.EnableOn with the relevant values.

Reply:
Done